projects/can_api/utils.py

```diff
@@ -1,14 +1,18 @@
 def get_rx_messages(subs, messages):
     """
-    Gets the received Messages from the list of messages, based on the subs 
+    Gets the received Messages from the list of messages, based on the subs
     dictionary.
     """
     rx_message_names = [d["name"] for d in subs]
     rx_messages = list(filter(lambda m: m.name in rx_message_names, messages))
+    masks = {
+        subscriber["name"]: subscriber["mask"] if "mask" in subscriber.keys() else 0x7FF
+        for subscriber in subs
+    }
```

Comment:
If there's a message in `subs` that isn't in our database's messages, it will still populate in the mask dictionary. The same thing happens with the mobs below. The C file will only generate based on the message names though, so the only consequence is some extra memory usage. 

@jack-greenberg  if this is something worth worrying about

Reply:
I don't think memory is a concern in this situation since it's just being used to generate a C File, and would just be thrown out if not in the messages, but I could be wrong.

Reply:
if the message doesn’t exist, we should throw an error and compiling should fail. can you implement that?

Reply:
yea, sounds good!